### ISSUE TITLE
Error: listen EADDRINUSE

### ISSUE THREAD

**Issue by GGICE** [thgh/rollup-plugin-serve#28]
Code as this:

```
serve({
  contentBase: [''],
  port: 8089
})
```
The port 8089 not used by other app, but can not start server. no port can use.

<img width="744" alt="screen shot 2018-12-20 at 8 17 49 pm" src="https://user-images.githubusercontent.com/3450419/50284576-5f2d8380-0494-11e9-8ae9-a551b7f3bcfd.png">

**Comment by sormy** [comment]
Do you have multiple inputs in rollup.config.js? Are you creating multiple instances of plugin? It is enough to have it included only once in any input definition.

**Comment by GGICE** [comment]
Yes I have multiple inputs in rollup.config.js, It was my fault ! sorry!
